BootloaderCorePkg/Tools/GenCapsuleFirmware.py

```diff
@@ -537,7 +538,7 @@ def ValidateRegistryFormatGuid(Argument):
     #
     # Create final capsule
     #
-    SignImage(Result, args.NewImage, args.HashType, args.PrivKey)
+    SignImage(Result, args.NewImage, args.HashType, args.SignScheme args.PrivKey)

```

Comment:
does this pass execution ?      Missing a  "," between  args.SignScheme and args.PrivKey.